components/Ripple/src/MDCRippleView.m

```diff
@@ -60,6 +60,15 @@ - (void)commonMDCRippleViewInit {
 - (void)layoutSubviews {
   [super layoutSubviews];
   [self updateRippleStyle];
+  self.frame = self.superview.bounds;
```

Comment:
Please be sure to call `CGRectStandardize` on the superview's bounds.

Reply:
done